src/machine/machinectl.c

```diff
@@ -110,8 +116,13 @@ typedef struct MachineInfo {
         const char *class;
         const char *service;
         char *os;
+        char *version_id;
 } MachineInfo;
 
+static inline const char *strdash_if_empty(const char *str) {
+        return isempty(str) ? "-" : str;
+}
```

Comment:
as mentioned, could you move this into src/basic/string-util.h, next to where strempty() and related calls are defined?